.github/workflows/pr-actions.yml

```diff
@@ -104,6 +104,7 @@ jobs:
       - run: npm install --omit=optional
       - run: |
           npm run check:links
+          npm run test-and-fix
```

Comment:
Is the previous step needed then?

Reply:
`test-and-fix` does much more than fixing the refcache (e.g. it also runs `fix:dict` and `fix:filenames`)

If I remember correctly `check:links` *should* update the refcache files except there is any issue going on.

Reply:
It doesn't appear to be updating anything, but `test-and-fix` does.

Reply:
that's odd ... but let's try it out then, I approved the PR